src/js/modules/forms/components/SectionForm.js

```diff
@@ -17,20 +25,28 @@ const styles = {
     padding: "5%",
 
   },
+  inputs: {
+    paddingLeft: "2%"
+  },
   titleInput: {
-    padding: "2%"
+    paddingBottom: "2%"
```

Comment:
Changed to margin. I'm using percents for now. If they become a problem, I'll switch to constants